cmd/eudico-stats/main.go

```diff
@@ -0,0 +1,138 @@
+package main
+
+import (
+	"context"
+	"net/http"
+	_ "net/http/pprof"
+	"os"
+	"time"
+
+	"github.com/filecoin-project/lotus/build"
+	lcli "github.com/filecoin-project/lotus/cli"
+	"github.com/filecoin-project/lotus/tools/stats/metrics"
+	"github.com/filecoin-project/lotus/tools/stats/sync"
+
+	logging "github.com/ipfs/go-log/v2"
+	"github.com/urfave/cli/v2"
+
+	"contrib.go.opencensus.io/exporter/prometheus"
+	stats "go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+)
+
+var log = logging.Logger("stats")
+
+func init() {
+	if err := view.Register(metrics.DefaultViews...); err != nil {
+		log.Fatal(err)
+	}
+}
+
+func main() {
+	local := []*cli.Command{
+		runCmd,
+		versionCmd,
+	}
+
+	app := &cli.App{
+		Name:    "eudico-stats",
+		Usage:   "Collect basic information about a filecoin network using lotus",
+		Version: build.UserVersion(),
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:    "lotus-path",
+				EnvVars: []string{"LOTUS_PATH"},
+				Value:   "~/.lotus", // TODO: Consider XDG_DATA_HOME
```

Comment:
Why `XDG_DATA_HOME`?

Reply:
I copied from the original code. Maybe I should just remove it.

Reply:
Maybe remove the comments (is confusing) and change `LOTUS_PATH` for `EUDICO_PATH` and use `~/.eudico` for the default location. This is the case for eudico